build/cqs.tsv

```diff
@@ -1,3 +1,4 @@
+asm:asm	3.1	compile
```

Comment:
it looks like spark-core is bringing this in via jersey-server. So it doesn't look like I did anything...